cmd/kops/create_cluster_integration_test.go

```diff
@@ -38,6 +39,13 @@ func TestCreateClusterMinimal(t *testing.T) {
 	runCreateClusterIntegrationTest(t, "../../tests/integration/create_cluster/minimal", "v1alpha2")
 }
 
+// TODO: https://github.com/kubernetes/kops/issues/2438
```

Comment:
Answered how to set a FF.  Though a FF might not even be necessary here, particularly if we have validation.